app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,27 @@
+class VotesController < ApplicationController
+  def upvote
+    @review = Review.find(params[:review_id])
+    @value = Vote.find_or_initialize_by(review: @review, user: current_user)
+    if @value.vote == 1
+      @value.vote -= 1
+    else
+      @value.vote = 1
+    end
+    @value.save
+    flash[:notice] = "Thanks for voting"
+    redirect_to bathroom_path(@review.bathroom)
+  end
+
+  def downvote
+    @review = Review.find(params[:review_id])
+    @value = Vote.find_or_initialize_by(review: @review, user: current_user)
+    if @value.vote == -1
+      @value.vote += 1
+    else
+      @value.vote = -1
+    end
+    @value.save
+    flash[:notice] = "Thanks for voting"
+    redirect_to bathroom_path(@review.bathroom)
+  end
+end
```

Comment:
You have a lot of duplication here. Consider this as something to refactor into a private method later on when you are polishing your app.